erpnext/templates/pages/projects.py

```diff
@@ -92,3 +94,6 @@ def get_timesheet_html(project, start=0):
 	return frappe.render_template("erpnext/templates/includes/projects/project_timesheets.html",
 		{"doc": {"timesheets": get_timesheets(project, start)}}, is_path=True)
 
+def get_attachments(doctype, project, start=0):
+	return frappe.get_all('File', filters= {"attached_to_name": project, "attached_to_doctype": doctype, "is_private":0},
+		fields=['name','file_name','file_url','is_private'])
```

Comment:
`name` is not being used in the template and `is_private` will always be 0 because of your filter.

Reply:
Also, the arguments are unnecessary, you need only pass `project`

Reply:
Done, thanks for your review